remo/settings/__init__.py

```diff
@@ -1,7 +1,9 @@
 import os
 
-from .base import *  # noqa
-
+try:
```

Comment:
Instead of try/except please explicitly check an env var and import accordingly.